ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/Divulgence.scala

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.tests
+
+import com.daml.ledger.api.testtool.infrastructure.{LedgerSession, LedgerTest, LedgerTestSuite}
+import com.daml.ledger.api.testtool.templates.{Divulgence1, Divulgence2}
+
+final class Divulgence(session: LedgerSession) extends LedgerTestSuite(session) {
```

Comment:
`DivulgenceIT` is "decorated" with quite a few comments. Please carry them over to this test as well. I think they are a great help as a preview of what the next few lines are about.
Having said that, the assertions in this test are super helpful as well.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/2282/commits/c7e6538d4efaacb3ed283c726acc0a12dfaaa80f